unittests/TreeStream/TreeStreamTest.cpp

```diff
@@ -19,7 +19,7 @@ TEST(TreeStreamTest, Basic) {
 
   std::vector<unsigned char> out;
   tsw.readStream(tos.getID(), out);
-  ASSERT_EQ(out.size(), 7);
+  ASSERT_EQ(out.size(), (unsigned) 7);
```

Comment:
Change the order of the argument (first is what you expect) and use appropriate values:

`ASSERT_EQ(7u, out.size());`